src/community/release/pom.xml

```diff
@@ -440,4 +403,74 @@
    </plugin>
   </plugins>
  </build>
+ 
+ <!-- profiles for individual modules -->
+ <profiles>
+   <profile>
+     <id>rest-ext</id>
+     <activation>
+       <property><name>rest-ext</name></property>
+     </activation>
+     <dependencies>
+       <dependency>
+         <groupId>org.geoserver.community</groupId>
+         <artifactId>gs-rest-ext</artifactId>
+         <version>${project.version}</version>
+       </dependency>
+     </dependencies>
+   </profile>
+   <profile>
+     <id>script</id>
+     <activation>
+       <property><name>script</name></property>
+     </activation>
+     <dependencies>
+       <dependency>
+         <groupId>org.geoserver.script</groupId>
+         <artifactId>gs-script-py</artifactId>
+         <version>${project.version}</version>
+       </dependency>
+       <dependency>
+         <groupId>org.geoserver.script</groupId>
+         <artifactId>gs-script-js</artifactId>
+         <version>${project.version}</version>
+       </dependency>
+       <dependency>
+         <groupId>org.geoserver.script</groupId>
+         <artifactId>gs-script-groovy</artifactId>
+         <version>${project.version}</version>
```

Comment:
As per the commented out lines above, script-js and script-groovy don't work at all, so this doesn't help for building script-py very much.

Reply:
No it does not, but it does place them all into the same profile.